### ISSUE TITLE
Upgrade to Storybook 7

### ISSUE THREAD

**Issue by benmccann** [AgnosticUI/agnosticui#261]
**Describe the bug**
I'd love to see Storybook upgraded to Storybook 7

**To Reproduce**
Go to `package.json` and see Storybook 6 is being used

**Expected behavior**
Use Storybook 7. It has built-in support for frameworks like Svelte and SvelteKit

**Screenshots**
N/A

**Desktop (please complete the following information):**
 - OS: any
 - Browser: any
 - Version: `master`

**Smartphone (please complete the following information):**
 - Device: any
 - OS: any
 - Browser: any
 - Version: `master`

**Issues with pull requests are deeply appreciated! But sometimes it's worth discussing first so the implementation works with AgnosticUI guidelines**

**Comment by roblevintennis** [comment]
Yes, the library basically got abandoned in about 2022/23 but I didn't have the heart to archive it. But, I'm just now trying to resurrect it. Apologies for the inconvenience and the new v2 I'm using is on storybook 9. It will be at parity soon'ish
